pytoml/writer.py

```diff
@@ -94,7 +97,8 @@ def dump(fout, obj):
             else:
                 fout.write('[{}]\n'.format(section_name))
 
-        for k in table:
+        table_keys = sorted(table.keys()) if sort_keys is True else table.keys()
```

Comment:
Please remove the `is True` part. There's a section about it in https://www.python.org/dev/peps/pep-0008/.